mypy.ini

```diff
@@ -0,0 +1,49 @@
+[mypy]
```

Comment:
I think we can move this configuration to `pyproject.toml` so it's all centralized.